app/services/google_calendar/event_data_service.rb

```diff
@@ -23,13 +23,14 @@ def datetime_parse(time, zone)
       end
 
       def normalize_whole_day_event(event)
-        event.start.date_time = Date.parse(event.start.date).beginning_of_day.to_datetime
-        event.end.date_time = Date.parse(event.end.date).beginning_of_day.to_datetime
+        [
```

Comment:
this looks interesting, so first you set values for the event and return it as a value, why ? :)